images/source/origin-local-release.repo

```diff
@@ -0,0 +1,4 @@
+[origin-local-release]
+name = OpenShift Origin Release from Local Source
+baseurl = file:///srv/origin-local-release/
```

Comment:
Since the imagebuilder is part of our build API, won't anyone who wants to build Origin already have the imagebuilder pulled locally?